Fix build failure: error: type 'QString' does not provide a call operator
https://github.com/QupZilla/qupzilla/issues/1293
https://github.com/QupZilla/qupzilla/commit/ca44302727bc216f102b89de828851deed418978
--- src/lib/app/mainapplication.cpp.orig
+++ src/lib/app/mainapplication.cpp
@@ -123,7 +123,7 @@ MainApplication::MainApplication(int &argc, char** argv)
     setOrganizationDomain("qupzilla");
 
 #ifdef Q_OS_MAC
-    DATADIR qApp->applicationDirPath() + QLatin1String("/../Resources/");
+    DATADIR = qApp->applicationDirPath() + QLatin1String("/../Resources/");
 #elif defined(Q_OS_UNIX) && !defined(NO_SYSTEM_DATAPATH)
     DATADIR = USE_DATADIR "/";
 #else
